src/components/problemDetailsModal/problemDetailsModal.tsx

```diff
@@ -46,15 +47,11 @@ const ProblemDetailsModalBody = (props: { problem: DetailsType | null }) => {
       </Flex>
       <Flex my="20px">
         <Card>
-          <Code colorScheme="grey">
-            {problem?.originalDetail
-              ? problem.originalDetail
-              : problem?.message}
-          </Code>
+          <Code colorScheme="grey">{problem?.originalDetail ? problem.originalDetail : problem?.message}</Code>
         </Card>
       </Flex>
       <Flex my="20px">
-        <Accordion>
+        <Accordion w="-webkit-fill-available">
```

Comment:
?

Is this a chakra feature?
I think something like "full" should be used instead.
Or maybe is mapped by chakra on the size prop.